tests/test_visualize.py

```diff
@@ -0,0 +1,40 @@
+"""Defines a dummy test."""
+
+from pathlib import Path
+
+import pytest
+
+from krecviz.visualize import visualize_krec
+
+BASE_PATH = Path(__file__).parent
+
+KREC_PATH = BASE_PATH / "assets" / "krec_examples" / "actuator_22_right_arm_shoulder_roll_movement.krec"
+URDF_PATH = BASE_PATH / "assets" / "urdf_examples" / "gpr" / "robot.urdf"
+
+
+@pytest.mark.slow
+def test_run_visualization(tmpdir: Path) -> None:
```

Comment:
Is this the test you are running to test the rotations?
You should have a test where you take an stl with three links and two joints, apply transformation and you test the position of the final link after the transformation explicitly.